server/main.lua

```diff
@@ -89,3 +89,13 @@ AddEventHandler('InteractSound_SV:PlayWithinDistance', function(maxDistance, sou
     TriggerClientEvent('InteractSound_CL:PlayWithinDistance', -1, source, maxDistance, soundFile, soundVolume)
   end
 end)
+
+RegisterNetEvent('InteractSound_SV:PlayWithinDistance')
+AddEventHandler('InteractSound_SV:PlayWithinDistance', function(maxDistance, soundFile, soundVolume)
+	local src = source
+    if maxDistance < 300 then
+	    TriggerClientEvent('InteractSound_CL:PlayWithinDistance', -1, GetEntityCoords(GetPlayerPed(src)), maxDistance, soundFile, soundVolume)
+    else
+        DropPlayer(src, 'Cheating!')
```

Comment:
How about instead of dropping the player here, we simply log a message in the console that a maximum distance is set. It would probably be best to make the 300 configurable, though, since I don't know all of the use cases for this script.